internal/manager/scheme/scheme.go

```diff
@@ -15,46 +17,53 @@ import (
 	incubatorv1alpha1 "github.com/kong/kubernetes-configuration/api/incubator/v1alpha1"
 )
 
-// Get returns a scheme aware of all types the manager can interact with.
-func Get() (*runtime.Scheme, error) {
-	scheme := runtime.NewScheme()
-
-	if err := apiextensionsv1.AddToScheme(scheme); err != nil {
-		return nil, err
-	}
-
-	if err := clientgoscheme.AddToScheme(scheme); err != nil {
-		return nil, err
-	}
+var (
+	kicScheme       = runtime.NewScheme()
+	addToSchemeDone atomic.Bool
+)
```

Comment:
If we want to move to a global, we could do something like:

```suggestion
var (
	kicScheme       = initScheme()
)
```

and return the global from `Get()`. We should only be aware that it might get problematic if we have tests that modify the scheme concurrently, as it will lead to data races.

Reply:
done.